src/persist-client/src/lib.rs

```diff
@@ -580,7 +580,6 @@ impl PersistClient {
 
     /// Test helper for a [Self::open] call that is expected to succeed.
     #[cfg(test)]
-    #[track_caller]
```

Comment:
will it always be the case that it doesn't have an effect. If it's meant to have an effect in the future it might be better to keep them and add an `allow`. cc @danhhz

Reply:
Hopefully not! https://github.com/rust-lang/rust/issues/87417 is the tracking issue for making it do something.

Reply:
Changed!